README.md

```diff
@@ -60,60 +60,67 @@ var smooch = new SmoochCore({
+| `appUsers.getAuthCode`        | Get auth code. | [:link:](https://docs.smooch.io/rest/?javascript#get-auth-code) |
+| `appUsers.getLinkRequests`    | Generate a transfer URL for a given channel type. | [:link:](https://docs.smooch.io/rest/?javascript#get-link-request) |
+| **menu Module**               | | |
+| `menu.get`                    | Get the specified app’s menu. | [:link:](https://docs.smooch.io/rest/?javascript#get-app-menu) |
+| `menu.configure`              | Configure the specified app’s menu. | [:link:](https://docs.smooch.io/rest/?javascript#update-app-menu) |
+| `menu.remove`                 | Remove the specified app’s menu. | [:link:](https://docs.smooch.io/rest/?javascript#delete-app-menu) |
+| **webhooks Module**           | | |
+| `webhooks.list`               | List all webhooks configured for a given app. | [:link:](https://docs.smooch.io/rest/?javascript#list-webhooks) |
+| `webhooks.create`             | Create a webhook for the specified app. | [:link:](https://docs.smooch.io/rest/?javascript#create-webhook) |
+| `webhooks.get`                | Get individual webhooks. | [:link:](https://docs.smooch.io/rest/?javascript#get-webhook) |
+| `webhooks.update`             | Update existing webhooks. | [:link:](https://docs.smooch.io/rest/?javascript#update-webhook) |
+| `webhooks.delete`             | Delete specified webhook. | [:link:](https://docs.smooch.io/rest/?javascript#delete-webhook) |
+| **apps Module**               | | |
+| `apps.create`                 | Create a new app. | [:link:](https://docs.smooch.io/rest/?javascript#create-app) |
+| `apps.list`                   | List all configured apps. | [:link:](https://docs.smooch.io/rest/?javascript#list-apps) |
+| `apps.get`                    | Get the specified app. | [:link:](https://docs.smooch.io/rest/?javascript#get-app) |
+| `apps.delete`                 | Delete the specified app, including all its enabled integrations. | [:link:](https://docs.smooch.io/rest/?javascript#delete-app) |
+| `apps.keys.create`            | Create a secret key for the specified app. | [:link:](https://docs.smooch.io/rest/?javascript#create-app-key) |
+| `apps.keys.list`              | List all secret keys for the sepcified app. | [:link:](https://docs.smooch.io/rest/?javascript#list-app-keys) |
+| `apps.keys.get`               | Get a secret key. | [:link:](https://docs.smooch.io/rest/?javascript#get-app-key) |
+| `apps.keys.delete`            | Delete a secret key. | [:link:](https://docs.smooch.io/rest/?javascript#delete-app-key) |
+| `apps.keys.getJwt`            | Get an app-scoped JWT signed using the requested keyId/secret pair. | [:link:](https://docs.smooch.io/rest/?javascript#get-app-jwt) |
+| **integrations Module**       | | |
+| `integrations.create`         | Create an integration endpoint to provision apps with front-end messaging channels. | [:link:](https://docs.smooch.io/rest/?javascript#create-integration) |
```

Comment:
This one is a bit weird. Why not just "Create a new integration"?

Reply:
I was using the wording from the original documentation as much as possible, but you are right. I'll make the change.

Reply:
ah I see where you took it from. The paraphrase was just a bit off (and the copy in the REST docs is not very good to begin with)

> The Create Integration endpoint allows you to provision apps with front-end messaging channels. See the sections below for channel specific instructions.

Using the original wording it would be something like

> Provision an app with a front-end messaging channel

but I still like "Create a new integration" over that